lib/mongoid_rails_migrations/active_record_ext/migrations.rb

```diff
@@ -332,6 +333,13 @@ def migrate
           migration.migrate(@direction)
           record_version_state_after_migrating(migration.version)
         rescue => e
+          output = Migration.buffer_output + "An error has occurred, #{migration.version} and all later migrations canceled:\n\n#{e}"
```

Comment:
Would be good to include the backtrace here (`e.backtrace.join("\n")`) so we get the full error like in the console.
Otherwise looks ok to me ;)